stable/coredns/README.md

```diff
@@ -0,0 +1,84 @@
+CoreDNS
+=======
+
+CoreDNS is a DNS server that chains middleware and provides Kubernetes DNS Services
+
+TL;DR;
+------
+
+```console
+$ helm install --name coredns stable/coredns
+```
+
+Introduction
+------------
+
+This chart bootstraps a [CoreDNS](https://github.com/coredns/coredns) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+This chart will provide DNS Services to a Kubernetes cluster and is a drop-in replacement for Kube/SkyDNS.
+
+Prerequisites
+-------------
+
+-	Kubernetes 1.4+ with Beta APIs enabled
+
```

Comment:
Probably should mention not to use it in Google Container Engine. See footnote here: https://blog.coredns.io/2017/03/01/coredns-for-kubernetes-service-discovery-take-2/

Unless you have a solution to that too 😄